packages/adaptive_scaffold/example/lib/main.dart

```diff
@@ -150,122 +211,108 @@ class _MyHomePageState extends State<MyHomePage>
+                // The conditional here is for navigation screens. The first
+                // screen shows the main screen and every other screeen shows
+                //  ExamplePage.
+                builder: (_) => (_navigationIndex == 0)
+                    ? Padding(
+                        padding: const EdgeInsets.fromLTRB(0, 32, 0, 0),
+                        child: _ItemList(
+                            selected: selected,
+                            items: allItems,
+                            selectCard: selectCard),
+                      )
+                    : const _ExamplePage()),
           },
         ),
         secondaryBody: _navigationIndex == 0
             ? SlotLayout(
                 config: <Breakpoint, SlotLayoutConfig?>{
                   Breakpoints.mediumAndUp: SlotLayout.from(
-                    // This overrides the default behavior of the secondaryBody
-                    // disappearing as it is animating out.
-                    outAnimation: AdaptiveScaffold.stayOnScreen,
-                    key: const Key('sBody'),
-                    builder: (_) =>
-                        _DetailTile(item: _all_Items[selected ?? 0]),
-                  ),
+                      // This overrides the default behavior of the secondaryBody
+                      // disappearing as it is animating out.
+                      outAnimation: AdaptiveScaffold.stayOnScreen,
+                      key: const Key('sBody'),
+                      builder: (_) =>
+                          _DetailTile(item: allItems[selected ?? 0]))
                 },
               )
             : null,
         bottomNavigation: SlotLayout(
           config: <Breakpoint, SlotLayoutConfig?>{
             Breakpoints.small: SlotLayout.from(
-              key: const Key('bottomNavigation'),
-              // You can define inAnimations or outAnimations to override the
-              // default offset transition.
-              outAnimation: AdaptiveScaffold.topToBottom,
-              builder: (_) => BottomNavigationBarTheme(
-                data: const BottomNavigationBarThemeData(
-                  unselectedItemColor: Colors.black,
-                  selectedItemColor: Colors.black,
-                ),
-                child: AdaptiveScaffold.toBottomNavigationBar(
-                    destinations: destinations),
-              ),
-            ),
+                key: const Key('bottomNavigation'),
+                // You can define inAnimations or outAnimations to override the
+                // default offset transition.
+                outAnimation: AdaptiveScaffold.topToBottom,
+                builder: (_) => BottomNavigationBarTheme(
+                    data: const BottomNavigationBarThemeData(
+                        selectedItemColor: Colors.black),
+                    child: AdaptiveScaffold.standardBottomNavigationBar(
+                        destinations: destinations)))
```

Comment:
Nit: add commas after destinations) and between the two last parentheses then format to fix indentation.